### ISSUE TITLE
[Feature] Customize the Cronjob image

### ISSUE THREAD

**Issue by sakazuki** [mariadb-operator/mariadb-operator#1097]
**Is your feature request related to a problem? Please describe.**

I'm facing an issue where I want to customize the `image` used in CronJob manifests generated by Backup manifests. Specifically, the current image value is hardcoded within the chart file ([here](https://github.com/mariadb-operator/mariadb-operator/blob/v0.36.0/deploy/charts/mariadb-operator/templates/configmap.yaml#L5)) and sourced from the MARIADB_OPERATOR_IMAGE environment variable in the ConfigMap/mariadb-operator-env. This rigid configuration prevents you from dynamically changing the image using the more flexible `values.yaml` file.

**Describe the solution you'd like**

To provide greater flexibility and customization, I would like to modify the [charts/mariadb-operator/templates/configmap.yaml](https://github.com/mariadb-operator/mariadb-operator/blob/v0.36.0/deploy/charts/mariadb-operator/templates/configmap.yaml) file to reference the image value directly from values.yaml. This would allow you to specify the desired image during the chart deployment process, rather than being limited to the hardcoded value.

**Describe alternatives you've considered**

**Environment details**:
- Kubernetes version: 1.30
- Kubernetes distribution: EKS
- mariadb-operator version: 0.36.0
- Install method: helm
- Install flavor: minimal

**Additional context**

**Comment by mmontes11** [comment]
I agree with @Sindrus , probably we should allow to fully customize the `ConfigMap`, see my comment:
- https://github.com/mariadb-operator/mariadb-operator/pull/1098#issuecomment-2585276299

**Comment by mmontes11** [comment]
This has been released in [0.37.0](https://github.com/mariadb-operator/mariadb-operator/releases/tag/0.37.0) 🚀 

Closing!
